app/src/main/java/de/tum/in/tumcampusapp/component/ui/studycard/CardsDetailActivity.java

```diff
@@ -58,14 +58,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     @Override
     protected StudyCard onLoadInBackground(Void... arg) {
-        /* if (!card.is_valid()) {
+        /* if (!cardView.is_valid()) {
             return null;
         }
         try {
             ChatMember chatMember = Utils.getSetting(this, Const.CHAT_MEMBER, ChatMember.class);
             final TUMCabeVerification v = TUMCabeVerification.create(this, chatMember);
             final Context c = this;
-            return TUMCabeClient.getInstance(c).addStudyCard(card, v);
+            return TUMCabeClient.getInstance(c).addStudyCard(cardView, v);
```

Comment:
This is still supposed to be card

Reply:
Anything related to study cards is currently not used in the app, so I did not bother with it. 
@kordianbruck can the study cards stuff be removed?